Markdown & Git

juliana-garden07 opened the pull request:
1. Task: https://github.com/rolling-scopes-school/tasks/blob/master/tasks/cv/git-markdown.md
2. Screenshot: 
![image](https://user-images.githubusercontent.com/54177417/132762730-9496b62f-9480-42bc-a27f-b6ba9c44b0c8.png)
3. Deploy: https://juliana-garden07.github.io/rsschool-cv/cv
4. Done: 10.09.21 / deadline 21.10.21
5. Score 100/100
-[x]Repository requirements are met (50/50)
-[x]Requirements for commits and Pull Request are met (50/50)

linegel commented on cv.md:
```diff
@@ -0,0 +1,23 @@
+###Junior Developer Resume
+1. **First Name, Last Name**: Juliana, Sadovskaya
+2. **Contact Info *(everal ways to contact you)***: juliana.sad07@gmail.com
```
It looks like too many stars for the closing of bold. It will not work

linegel replied:
Looks like it's not only here, but everywhere you use *** instead of two for closing of **bold**.  It should be fixed before merge

linegel commented on cv.md:
````diff
@@ -0,0 +1,23 @@
+4. **Skills *(e.g. programming languages, frameworks, methodologies, version control, tools etc.)***: CSS, HTML, JavaScript
+5. **Code examples *(LATEST)***: 
+    
+    ```javascript
````
Do you really need there two spaces? Is it follows specification?

linegel commented on cv.md:
```diff
@@ -4,8 +4,7 @@
 3. **Summary *(your goal, wishes, reveal what is important for you, what do you want and why. Some kind of self-presentation. In case of lack of experience  Junior Developer sells his/her potential, his/her passion and ability to learn fast. You shouldn't think that everybody is going to teach you when you come to the workplace. Rather being a Junior means always learning new things from everywhe ...
```
You know what? I just understand why I was so surprised to see questions from rsschool in bold -- because it's mich more intuitive to have YOU answers as bold, not their questions.

juliana-garden07 committed:
init: start cv project
*add name
*add contact info
*add summary
*add skills

juliana-garden07 committed:
feat: add new points
*add code examples
*add experience

juliana-garden07 committed:
feat: add new points
*add education
*add english

juliana-garden07 committed:
docs: update readme file

juliana-garden07 committed:
docs: delete readme

juliana-garden07 committed:
init: start cv task
*create html file
*create css file
*add normalize.css file
*add image and icons

juliana-garden07 committed:
feat: add next features
*add adaptabylity for 4 sizes of devise
*add semantic tegs
*add burger-menu

juliana-garden07 committed:
fix: fix adaptability for small devises

juliana-garden07 committed:
feat: add some features
*add sintax hightlighting
*add some link to sotial networks
*fix adaptability

juliana-garden07 committed:
fix: fix some problems
*fix header teg
*fix menu
*fix adaptability

juliana-garden07 committed:
fix: fix width menu

juliana-garden07 committed:
Merge pull request #2 from juliana-garden07/rsschool-cv-html

HTML, CSS & Git Basics

juliana-garden07 committed:
fix: fix adaptability

juliana-garden07 committed:
Merge pull request #4 from juliana-garden07/cv-html-css

fix: fix adaptability
